ChromeExtension/js/apikeys.js

```diff
@@ -1,58 +1,53 @@
-document.addEventListener('DOMContentLoaded', function () {
+function apiKeyViewModel() {
     if (localStorage["apiKey"] == null && localStorage["apiSecret"] == null) {
         localStorage.setItem("apiKey", "");
         localStorage.setItem("apiSecret", "");
 
     }
-
-    $(document).ready(function() {
-        $('#tutorial').on('click', 'a', function () {
-            chrome.tabs.create({
-                url: $(this).attr('href')
-            });
-            return false;
-        });
-
-    });
-
- 
-
-    document.getElementById("apiKey").value = localStorage["apiKey"];
-    document.getElementById("apiSecret").value = localStorage["apiSecret"];
-
-    if (localStorageHasValue("defaultGroup")) {
-        $("#help").css("visibility", "hidden");
-        $("#back").css("visibility", "visible");
-    } else {
-        $("#back").css("visibility", "hidden");
-        $("#help").css("visibility", "visible");
-    }
-
-
-    var saveButton = document.getElementById('saveButton');
-
-    apiKey.addEventListener('change', function() {
+        apiKey.addEventListener('change', function () {
         var apiKey = document.getElementById("apiKey").value;
         localStorage.setItem("apiKey", apiKey);
     });
 
-    apiSecret.addEventListener('change', function() {
+    apiSecret.addEventListener('change', function () {
         var apiKey = document.getElementById("apiSecret").value;
         localStorage.setItem("apiSecret", apiKey);
     });
 
-    // onClick funcion
-    saveButton.addEventListener('click', function() {
+    var self = this;
+    self.apiKey = ko.observable(localStorage["apiKey"]);
+    self.apiSecret = ko.observable(localStorage["apiSecret"]);
+    self.showBackLink = ko.observable(false);
+    self.showHelpLink = ko.observable(true);
+    self.loadKey = function () {
+        var ak = localStorage["apiKey"];
+        if (ak == '') {

```

Comment:
Make that into a method to check for empty so we don't copy the equality everywhere also use  === instead of ==